pynicotine/core.py

```diff
@@ -464,8 +464,8 @@ def _check_privileges(self, msg):
         days = hours // 24
 
         if msg.seconds == 0:
-            log.add(_("You have no Soulseek privileges. Privileges are not required, but allow your downloads "
-                      "to be queued ahead of non-privileged users."))
+            log.add(_("You have no Soulseek privileges. While your privileges are active, your downloads "
+                      "will be queued ahead of those of non-privileged users."))
```

Comment:
`While your privileges are active` -> `If you have privileges`

Reply:
That's less clear. I don't mind spending a word on that.

Reply:
It just seems contradictory... "_while_ your privileges _are active_" implies that you currently have privileges, yet in the previous sentence it is stated that you don't.

The proposed string is too ambiguous.